.github/workflows/tests.yml

```diff
@@ -8,27 +8,27 @@ jobs:
         python-version:
           - "3.8"
           - "3.9"
-        django-version:
-          - "3.2"
-          - "4.0"
-          - "4.1"
-          - "4.2"
+          - "3.10"
+          - "3.11"
+          - "3.12"
 
     steps:
       - uses: actions/checkout@v3
+
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
-      - name: Install dependencies (django ${{ matrix.django-version }})
+
+      - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
-          pip install flake8 pyyaml coveralls pytest-django django==${{ matrix.django-version }}.*
-          pip install -e .
+          pip install flake8
+
       - name: Lint with flake8
         run: |
           # stop the build if there are Python syntax errors or undefined names
           flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
flake8 could be split into a separate job but for now I left it on each environment.